server/pulp/server/db/migrations/lib/managers.py

```diff
@@ -77,3 +86,141 @@ def find_with_importer_type(importer_type_id):
             results.append(repo)
 
         return results
+
+
+class UserManager(object):
+
+    def ensure_admin(self):
+        """
+        This function ensures that there is at least one super user for the system.
+        If no super users are found, the default admin user (from the pulp config)
+        is looked up or created and added to the super users role.
+        """
+        role_manager = factory.role_manager()
+        if self.get_admins():
+            return
+
+        default_login = config.config.get('server', 'default_login')
+
+        admin = get_collection('users').find_one({'login': default_login})
```

Comment:
Since this is in the migrations library, I want it to be as independent of the production code as possible and as similar to the old code as possible. This is *almost* identical to the manager code that was removed, so I believe that doing it this way will have the least risk. If we use the mongoengine model code and we change the model, we may inadvertently also change how the database is initialized. If we initialize in the way that we used to, then any changes should be reflected in the migrations. The idea is that once in the migrations lib, the code should not need to change and core code changes shouldn't affect it either.